Sources/TensorFlow/Layers/Dropout.swift

```diff
@@ -73,3 +73,68 @@ public struct Dropout<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
     }
   }
 }
+
+/// A gaussian noise layer
+///
+/// Gaussian Noise adds noise sampled from a normal distribution with mean zero
+public struct GaussianNoise<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
+  @noDerivative public let standardDeviation: Tensor<Scalar>
+  
+  /// Creates a Gaussian noise layer
+  ///
+  /// - Parameter standardDeviation: Standard deviation of the Guassian distribution
+  public init(standardDeviation: Scalar) {
+    self.standardDeviation = Tensor<Scalar>(standardDeviation)
+  }
+  
+  /// Returns output obtained by adding noise
+  /// - Parameter input: The input to the layer
+  /// - Returns: The output
```

Comment:
```suggestion
  /// Returns a tensor obtained by adding noise to `input`
```

I think that the extra comments don't substantially help the reader, so I think eliding them is probably better. Thanks!